### ISSUE TITLE
Github workflow builds fail with wasm-bindgen 0.2.96

### ISSUE THREAD

**Issue by ajrcarey** [ajrcarey/pdfium-render#177]
During regular comment update to #137, Github workflow build failed with error 

```
`#![feature]` may not be used on the stable release channel
 --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-macro-0.2.96/src/lib.rs:4:5
  |
4 |     feature(allow_internal_unstable),
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by ajrcarey** [comment]
The problem is temporarily resolved by locking the `wasm-bindgen` dependency in Cargo.toml to 0.2.95.

**Comment by ajrcarey** [comment]
Added notes to `Cargo.toml` and `README.md`. Change will be included in release 0.8.27.

**Comment by ajrcarey** [comment]
Failed build: https://github.com/ajrcarey/pdfium-render/actions/runs/12089789793

**Comment by ajrcarey** [comment]
Whatever the problem was, it was corrected by the time of `wasm-bindgen` 0.2.97. Removed version constraint in `Cargo.toml` as recommended in https://github.com/ajrcarey/pdfium-render/pull/191 .
